src/events/Windows.cpp

```diff
@@ -216,6 +216,20 @@ void Events::listener_mapWindow(void* owner, void* data) {
             } else {
                 Debug::log(ERR, "Rule idleinhibit: unknown mode %s", IDLERULE.c_str());
             }
+        } else if (r.szRule.find("bordercolor") == 0) {
+            try {
+                std::string colorPart = r.szRule.substr(r.szRule.find_first_of(' ') + 1);
+
+                if (colorPart.contains(' ')) {
```

Comment:
ummm, comma? Then why are you looking for `' '`?

Reply:
comment was wrong, we are looking for a space to distinguish between one and to args for the rule